CodeGeneration/Sources/generate-swiftsyntax/GenerateSwiftSyntax.swift

```diff
@@ -129,8 +128,12 @@ struct GenerateSwiftSyntax: ParsableCommand {
         ),
         GeneratedFileSpec(swiftSyntaxBuilderGeneratedDir + ["RenamedChildrenBuilderCompatibility.swift"], renamedChildrenBuilderCompatibilityFile),
       ]
-      + BASE_KIND_FILES.map { baseKind in
-        GeneratedFileSpec(swiftSyntaxGeneratedDir + ["syntaxNodes", baseKind.value], syntaxNode(emitKind: baseKind.key))
+      // This split of letters produces files for the syntax nodes that have about equal size, which improves compile time
+      + ["AB", "C", "D", "EF", "GHI", "JKLMN", "OP", "QRS", "TUVWXYZ"].flatMap { letters in
+        [
+          GeneratedFileSpec(swiftSyntaxGeneratedDir + ["syntaxNodes", "SyntaxNodes\(letters).swift"], syntaxNode(nodesStartingWith: Array(letters))),
+          GeneratedFileSpec(swiftSyntaxGeneratedDir + ["raw", "RawSyntaxNodes\(letters).swift"], rawSyntaxNodesFile(nodesStartingWith: Array(letters))),
+        ]
```

Comment:
Nice idea! 
Firstly I was "Why not each type get their own file" and I came across the cmake file 😬

Reply:
I'm not that much into how the invitation of the Swift frontend work. 
But thanks for the explanation! 🙌